content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -112,35 +112,49 @@ documentation for the plugins about what port(s) those need.
 
 ## Installing runtime {#installing-runtime}
 
-Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+To run containers in Pods, Kubernetes uses a
+{{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
 
-Since v1.14.0, kubeadm will try to automatically detect the container runtime on Linux nodes
-by scanning through a list of well known domain sockets. The detectable runtimes and the
-socket paths, that are used, can be found in the table below.
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux nodes" %}}
+
+By default, Kubernetes uses the
+{{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
+to interface with your chosen container runtime.
+
+kubeadm automatically tries to detect the container runtime
+by scanning through a list of well known Unix domain sockets. The detectable
+runtimes and the socket paths that are tried are:
 
 | Runtime    | Domain Socket                    |
 |------------|----------------------------------|
 | Docker     | /var/run/docker.sock             |
 | containerd | /run/containerd/containerd.sock  |
 | CRI-O      | /var/run/crio/crio.sock          |
 
+<br />
```

Comment:
This is a bit of a bodge to add space after the table. Hope it's acceptable. If not I have to look at CSS.

Reply:
@sftim , I guess the table in the tab-UI lacks a margin, top and bottom. I _think_ the ```<br/>``` is OK. Do you want to add a ```<br/>``` before the table as well?

Reply:
Think it's OK without space before. If I wanted more space top and bottom I think that'd be time to crack open the CSS.